_data/accessibility-tests/footer.yml

```diff
@@ -0,0 +1,97 @@
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.4
+  - summary: Form field instructions are visible.
+    summary_additional: When viewing a form field in the big footer, instructions for entering text are visible.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.3.2
+# Zoom/screen magnification tests
+  - summary: Footer remains visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 200%, you can still see footer text and it doesn't overlap with or get cut off by any other content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.4
+# Keyboard navigation tests
+  - summary: Keyboard navigates through all elements in the footer.
+    summary_additional: When you navigate through the footer with a keyboard, you can interact with every element inside the footer.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.1
+  - summary: Footer does not trap keyboard focus.
+    summary_additional: When you navigate with a keyboard, you can easily move into and out of the footer.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.2
+  - summary: Focus indicator is clearly visible.
+    summary_additional: When you use a keyboard to focus on footer elements, there will be a visible outline or other clear indication of focus. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.7
+# Screen reader tests
+  - summary: Screen reader announces image alt text.
+    summary_additional: When you navigate through the footer using a screen reader, you hear descriptions for any important images or graphics.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.1.1
+  - summary: Screen reader announces footer navigation or region.
+    summary_additional: When using a screen reader to acces the footer, you hear "footer navigation, navigation landmark (or region in JAWS). You can also use shortcut keys in the screen reader to navigate the footer. 
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.1
+  - summary: Screen reader announces cue to enter email. 
+    summary_additional: When using a screen reader to access the sign up form field in the big footer, you hear a prompt to enter your email address. 
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.1
+  - summary: Screen reader focus follows logical content order.
+    summary_additional: When you navigate through the footer with a screen reader, the focus stays in order of the content as intended. 
+    test_status: pass
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.3.2
+  - summary: Screen reader announces form buttons.
```

Comment:
@amycole501 @alex-hull @mahoneycm 

**suggestion (non-blocking):** Recommend we name the applicable variant in the summary for clarity. Also recommend we borrow from the 4.1.2 item on the button tests page for simplicity. 
```suggestion
  - summary: In the big footer variant, the screen reader announces the button role.
```

Reply:
I'm glad you're calling out the variants; I have them separated in the spreadsheet but I like how you've incorporated the "big footer" callout.